schema/defs/vrs/Genotype.rst

```diff
@@ -0,0 +1,34 @@
+**Computational Definition**
+
+A set of trans-phased :ref:`MolecularVariation` members, with associated copy counts, across a specified number of genomic locus `copies`.
+
+**Information Model**
+
+Some Genotype attributes are inherited from :ref:`Variation`.
+
+.. list-table::
+   :class: clean-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+   
+   *  - Field
+      - Type
+      - Limits
+      - Description
+   *  - _id
+      - :ref:`CURIE`
+      - 0..1
+      - Variation Id. MUST be unique within document.
+   *  - type
+      - string
+      - 1..1
+      - MUST be "Genotype"
+   *  - members
+      - :ref:`GenotypeMember`
+      - 1..m
+      - Each GenotypeMember in `members` describes a :ref:`MolecularVariation` and the count of that variation at the locus.
+   *  - copies
```

Comment:
Are two Genotypes the same if their `members` are identical but their `copies` are not?

My recommendation would be to drop the copies for now unless you know that you have a use for them.

Reply:
No. If the copies are different in any way then they are not precisely the same genotypes. If we are trying to acheive the level of exactness as we do for Alleles then we are asserting that the _ploidy_ or `copies` of the _in-trans_ instances of the underlying loci that covers the region containing the members is not the same across two different Genotypes (even if the members are identical) then it is not considered an exact match.

We believe this adheres to the VRS principles of precision.